native/CMakeLists.txt

```diff
@@ -102,7 +102,7 @@ if(OPENHARMONY AND OHOS)
 endif()
 
 ## disable videoplay on non-mobile platforms
-if(NOT ANDROID AND NOT IOS AND NOT OHOS)
+if(NOT ANDROID AND NOT IOS AND NOT OHOS AND NOT OPENHARMONY)
     set(USE_VIDEO OFF)
     set(USE_WEBVIEW OFF)
 endif()
```

Comment:
As OPENHARMONY implement it, why need to set USE_WEBVIEW off?

Reply:
Disable videoplay on non-mobile platforms.OpenHarmony is the mobile platform